packages/js/src/plugins/auctionHouseModule/operations/withdrawFromBuyerAccount.ts

```diff
@@ -0,0 +1,227 @@
+  response: SendAndConfirmTransactionResponse;
+};
+
+/**
+ * @group Operations
+ * @category Handlers
+ */
+export const withdrawOperationHandler: OperationHandler<WithdrawOperation> = {
+  handle: async (operation: WithdrawOperation, metaplex: Metaplex) =>
+    withdrawFromBuyerAccountBuilder(operation.input, metaplex).sendAndConfirm(
+      metaplex,
+      operation.input.confirmOptions
+    ),
+};
+
+// -----------------
+// Builder
+// -----------------
+
+/**
+ * @group Transaction Builders
+ * @category Inputs
+ */
+export type WithdrawBuilderParams = Omit<WithdrawInput, 'confirmOptions'> & {
+  instructionKey?: string;
+};
+
+/**
+ * @group Transaction Builders
+ * @category Contexts
+ */
+export type WithdrawBuilderContext = Omit<WithdrawOutput, 'response'>;
+
+/**
+ * Withdraws funds from the user's buyer escrow account to the given auction house.
+ *
+ * ```ts
+ * const transactionBuilder = metaplex
+ *   .auctionHouse()
+ *   .builders()
+ *   .withdrawFromBuyerAccountBuilder({ auctionHouse, buyer, amount });
+ * ```
+ *
+ * @group Transaction Builders
+ * @category Constructors
+ */
+export const withdrawFromBuyerAccountBuilder = (
+  params: WithdrawBuilderParams,
+  metaplex: Metaplex
+): TransactionBuilder<WithdrawBuilderContext> => {
+  const { auctionHouse, auctioneerAuthority, amount } = params;
+
+  if (auctionHouse.hasAuctioneer && !params.auctioneerAuthority) {
+    throw new AuctioneerAuthorityRequiredError();
+  }
+
+  const amountBasisPoint = amount.basisPoints;
+  const buyer = params.buyer ?? (metaplex.identity() as Signer);
+  const authority = params.authority ?? auctionHouse.authorityAddress;
+  const escrowPayment = findAuctionHouseBuyerEscrowPda(
+    auctionHouse.address,
+    toPublicKey(buyer)
+  );
+
+  // Accounts,
+  const accounts: WithdrawInstructionAccounts = {
+    wallet: toPublicKey(buyer),
+    receiptAccount: toPublicKey(buyer),
+    escrowPaymentAccount: escrowPayment,
+    treasuryMint: auctionHouse.treasuryMint.address,
+    authority: toPublicKey(authority),
+    auctionHouse: auctionHouse.address,
+    auctionHouseFeeAccount: auctionHouse.feeAccountAddress,
+  };
+
+  // Args.
+  const args = {
+    escrowPaymentBump: escrowPayment.bump,
+    amount: amountBasisPoint,
+  };
+
+  // Withdraw Instruction.
+  let withdrawInstruction = createWithdrawInstruction(accounts, args);
+  if (auctioneerAuthority) {
+    const ahAuctioneerPda = findAuctioneerPda(
+      auctionHouse.address,
+      auctioneerAuthority.publicKey
+    );
+
+    const accountsWithAuctioneer = {
+      ...accounts,
+      auctioneerAuthority: auctioneerAuthority.publicKey,
+      ahAuctioneerPda,
+    };
+
+    withdrawInstruction = createAuctioneerWithdrawInstruction(
+      { ...accountsWithAuctioneer },
+      args
+    );
+  }
+
+  // Signers.
+  const withdrawSigners = [buyer, authority, params.auctioneerAuthority].filter(
+    isSigner
+  );
+
+  return (
+    TransactionBuilder.make()
+
```

Comment:
Also, I would add 

**.setFeePayer(payer)** to allow another fee payer on a transaction similarly to other operations.